app/1.0.x/upgrading.md

```diff
@@ -13,6 +13,41 @@ when upgrading, as well as take you through the correct sequence of steps
 in order to obtain a **no-downtime** migration in different upgrade
 scenarios.
 
+## Zero-downtime migrations
```

Comment:
This section already exists below in the same document: `Migration Steps from 0.14`. Please remove this addition from this PR